torchgeo/datamodules/inria.py

```diff
@@ -243,3 +245,7 @@ def on_after_batch_transfer(
         if "mask" in batch:
             batch["mask"] = rearrange(batch["mask"], "b () h w -> b h w")
         return batch
+
+    def plot(self, *args: Any, **kwargs: Any) -> plt.Figure:
+        """Run :meth:`torchgeo.datasets.InriaAerialImageLabelingDataModule.plot`."""
+        return self.dataset.plot(*args, **kwargs)
```

Comment:
@adamjstewart Simplified the plot function as suggested in the object detection PR.